### ISSUE TITLE
Some OncoKB variants are missing in the TEMPO filtered maf

### ISSUE THREAD

**Issue by ShwetaCh** [mskcc/tempo#776]
While looking into the WES-recapture project, I tried to find evidence for IMPACT variants in WES and found that some were missing inspite of sufficient coverage/quality.

Details below:
Out of 326, there are 102 with VAF > 5% and 70 with VAF > 10% that are missing. For each of these variants I have tracked them to see if they were present in the unfiltered maf and then if still not found, I traced them to the VCF, and if still not found, I've tried to reason them myself looking at the exome_config file which dictates the read depth and other filter thresholds. I've documented the reasons for missing based on my investigation in a google sheet and happy to share that sheet. 

In summary, out of those 326, 80 of them missing were not explained by any FILTER tag in maf or vcf or by the filter criteria in the exome_config. This is a very high FN rate.

On spot checking these variants were present in Roslin, and when reran with new version of TEMPO, they were still not found (based on spot checking only). So the issue does not seem to be version specific.

**Comment by gongyixiao** [comment]
Is this the same as this issue https://github.com/mskcc/tempo/issues/922?
